src/core/openthread_freertos.c

```diff
@@ -94,11 +95,6 @@ void otrTaskNotifyGiveFromISR()
     BaseType_t taskWoken;
 
     vTaskNotifyGiveFromISR(sMainTask, &taskWoken);
-
```

Comment:
Removing `portEND_SWITCHING_ISR` will postpone context switch to next tick, making it possible to call `taskNotify` anywhere in ISR service routine. Since real-time issue is not critical for this project, I think it's acceptable.